gradle/versions.gradle

```diff
@@ -12,6 +12,7 @@ ext {
     androidMavenGradlePluginVersion = "1.4.1"
 
     // Libraries
+    androidxTestCoreVersion = "1.2.0"
```

Comment:
This constants will likely need to be added `android-complete` too -- since AFAIK these values are always resolved from the root project

Reply:
Already done and merged:
https://github.com/AzureAD/android-complete/pull/18